### PR TITLE
A **LOT** of clean-up in `examples/Notebook`

### PR DESCRIPTION
**Pull request by jdfreder**
@minrk sorry this took so long, but it seemed to need a lot of work.

### CODE REVIEW

**Comment by jdfreder** [comment]
- also, I cleared all of the output so the notebooks are in a fresh state, for nbviewer we may want to run some of them (but most of the content here isn't code centric).

File: examples/Notebook/Configuring the Notebook and Server.ipynb [hunk 1]
```diff
@@ -371,7 +287,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.4.2"
+   "version": "3.4.3"
```
Comment (Carreau) [inline examples/Notebook/Configuring the Notebook and Server.ipynb]: Pffff, so last week.
```
$ ipython
Python 3.5.0a1+ (default, Feb 25 2015, 13:39:51)
```

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb [hunk 2]
```diff
@@ -50,230 +46,105 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We will be converting `Custom Display Logic.ipynb`. \n",
-    "Be sure to have runed some of the cells in it to have output otherwise you will only see input in nbconvert.\n",
-    "Nbconvert **do not execute the code** in the notebook files, it only converts what is inside."
+    "As a test, the `Index.ipynb` notebook in the directory will be convert. \n",
+    "\n",
+    "If you're converting a notebook with code in it, make sure to run the code cells that you're interested in before attempting to convert the notebook.  Nbconvert **does not execute the code cell** of the notebooks that it converts."
```
Comment (Carreau) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Well it **can** run code, but I guess that's a good first order approx.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb [hunk 3]
```diff
@@ -382,7 +199,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "I'll let you read Jinja manual for the exact syntax of the template."
+    "For details about the template syntax, refer to [Jinja's manual](http://jinja.pocoo.org/docs/dev/)."
```
Comment (Carreau) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: `http://jinja2.readthedocs.org/en/latest/intro.html` ? you link to developement version of the doc.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb [hunk 4]
```diff
@@ -396,22 +213,22 @@
...
+    "The notebook file format supports attaching arbitrary JSON metadata to each cell.  Here, as an exercise, you will use the metadata to tags cells."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "First you need to choose another notebook you want to convert to html, and tag some of the cell with metadata.\n",
-    "You can see the file `soln/celldiff.js` for a solution on how to tag, or follow the javascript tutorial to see how to do that. Use what we have written there to tag cells of some notebooks to `Easy`|`Medium`|`Hard`|`<None>`, and convert this notebook using your template."
+    "First you need to choose another notebook you want to convert to html, and tag some of the cell with metadata.  You can refere to the file `soln/celldiff.js` as an example or follow the Javascript tutorial to figure out how do change cell metadata.  Assuming you have a notebook with some of the cells tagged as `Easy`|`Medium`|`Hard`|`<None>`, the notebook can be converted specially using a custom template.  Design your remplate in the cells provided below.\n",
```
Comment (Carreau) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Some of the cell**s** ? or some of the cell ? (real question)

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb [hunk 5]
```diff
@@ -472,10 +289,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As of all of IPython nbconvert can be configured using profiles and passing the `--profile` flag. \n",
-    "Moreover if a `config.py` file exist in current working directory nbconvert will use that, or read the config file you give to it with the `--config=<file>` flag. \n",
-    "\n",
-    "In the end, if you are often running nbconvert on the sam project, `$ ipython nbconvert` should be enough to get you up and ready."
+    "IPython nbconvert can be configured using the default profile and by selecting a non-default a profile via `--profile` flag.  Additionally, if a `config.py` file exist in current working directory, nbconvert will use that as config."
```
Comment (Carreau) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: `and by selecting a non-default a profile`  seem weird.

File: examples/Notebook/Notebook Basics.ipynb [hunk 6]
```diff
@@ -11,57 +11,125 @@
...
+   "metadata": {},
+   "source": [
+    "You can start the notebook server from the command line (Terminal on Mac/Linux, CMD prompt on Windows) by running the following command: \n",
     "\n",
     "    ipython notebook\n",
     "\n",
-    "For more details on how to run the notebook server, see [Running the Notebook Server](Running the Notebook Server.ipynb)."
+    "This will print some information about the notebook server in your terminal, including the URL of the web application (by default, `http://127.0.0.1:8888`). It will then open your default web browser to this URL.\n",
+    "\n",
+    "When the notebook opens, you will see the **notebook dashboard**, which will show a list of the notebooks and subdirectories in the directory where the notebook server was started (as seen in the next section, below). Most of the time, you will want to start a notebook server in the highest directory in your filesystem where notebooks can be found. Often this will be your home directory."
```
Comment (Carreau) [inline examples/Notebook/Notebook Basics.ipynb]: Suggestion : list of notebooks, **files**, and directory.

File: examples/Notebook/Notebook Basics.ipynb [hunk 7]
```diff
@@ -11,57 +11,125 @@
...
-    "To create a new notebook, click on the \"New Notebook\" button at the top of the list.\n",
+    "Or start notebook server without opening a web browser.\n",
     "\n",
-    "Notebooks can be uploaded to the current directory by dragging a notebook file onto the notebook list or by the \"click here\" text above the list.\n",
+    "    ipython notebook --no-browser\n",
     "\n",
-    "The notebook list shows a red \"Shutdown\" button for running notebooks and a \"Delete\" button for stopped notebooks. Notebook remain running until you explicitly click the \"Shutdown\" button; closing the notebook's page is not sufficient.\n",
+    "The notebook server has a number of other command line arguments that can be displayed with the `--help` flag: \n",
     "\n",
-    "To see all of your running notebooks along with their directories, click on the \"Running\" tab:\n",
```
Comment (Carreau) [inline examples/Notebook/Notebook Basics.ipynb]: I don't think this part on many notebook server is useful anymore.

Comment (Carreau) [inline examples/Notebook/Notebook Basics.ipynb]: I'll make a break on my review here, and will continue later.

File: examples/Notebook/Configuring the Notebook and Server.ipynb (hunk 1 above)
Comment (jdfreder) [inline examples/Notebook/Configuring the Notebook and Server.ipynb]: :)

File: examples/Notebook/Notebook Basics.ipynb (hunk 7 above)
Comment (jdfreder) [inline examples/Notebook/Notebook Basics.ipynb]: > I don't think this part on many notebook server is useful anymore.

The part I already removed, right?



> I'll make a break on my review here, and will continue later.

Okay, thanks!  I know it's a lot to review.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb (hunk 2 above)
Comment (jdfreder) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Oh that's right, I forgot about that. - But it doesn't without being requested.  I'll change the wording so it's not as aggressive.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb (hunk 3 above)
Comment (jdfreder) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Thanks!  I just linked what Google landed me on.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb (hunk 4 above)
Comment (jdfreder) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Some of the cell**s**, sorry I missed that typo.

File: examples/Notebook/Notebook Basics.ipynb (hunk 7 above)
Comment (Carreau) [inline examples/Notebook/Notebook Basics.ipynb]: No the `You can start more than one notebook server at the same time. By default, the first ...` and all this stuff about port. This was mostly, IMHO, when people were using many servers for many dirs and/or languages.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb (hunk 5 above)
Comment (jdfreder) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Yeah that is weird sounding.  I changed the wording a bit.

**Comment by jdfreder** [comment]
@Carreau thanks for the comments.  They should all be addressed in the last commit, except for your last comment, the one I didn't understand.

File: examples/Notebook/Converting Notebooks With nbconvert.ipynb (hunk 4 above)
Comment (takluyver) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Also, I spot the word 'remplate'

Comment (jdfreder) [inline examples/Notebook/Converting Notebooks With nbconvert.ipynb]: Thanks , last commit should fix it.

File: examples/Notebook/Notebook Basics.ipynb (hunk 7 above)
Comment (jdfreder) [inline examples/Notebook/Notebook Basics.ipynb]: Ahh I see.  I removed the multiple server bit.

**Comment by jdfreder** [comment]
Comments addressed.

**Comment by minrk** [comment]
Okay to merge this one? comments seem addressed. I'm not sure about the removal of all the output, but I think it's okay for now.

**Comment by jdfreder** [comment]
Yeah it's ready.

**Pull request merged**